sdk/data/aztables/internal/connection.go

```diff
@@ -25,7 +25,7 @@ func NewConnection(endpoint string, options *azcore.ClientOptions) *Connection {
 	if options != nil {
 		cp = *options
 	}
-	return &Connection{u: endpoint, p: runtime.NewPipeline(module, version, nil, nil, &cp)}
+	return &Connection{u: endpoint, p: runtime.NewPipeline(moduleName, moduleVersion, runtime.PipelineOptions{}, &cp)}
```

Comment:
Note that we no longer generate this, it was just the easiest way to get this working.  We should remove it at some point.